lib/worker.go

```diff
@@ -220,16 +220,14 @@ func (w *DefaultBaseWorker) doPreInit(ctx context.Context) error {
 		return errors.Trace(err)
 	}
 
+	if err := w.statusSender.SafeSendStatus(ctx, WorkerStatus{Code: WorkerStatusInit}); err != nil {
```

Comment:
what does the workerstatusInit mean to the worker?

Reply:
Keep the original logic, after InitImpl is called, change worker status to `WorkerStatusInit`. But this needs to be refactored later, since user could have updated status in `InitImpl`, then the worker status will be overwritten in `doPostInit`